apps/dashboard/components/logs/queries/queries-method-row.tsx

```diff
@@ -0,0 +1,24 @@
+import type { LogsFilterUrlValue } from "@/app/(app)/logs/filters.schema";
+import { Conversion } from "@unkey/icons";
+import { QueriesPill } from "./queries-pill";
+
+type MethodRowProps = {
+  methods: LogsFilterUrlValue[] | null;
+};
+export const MethodRow = ({ methods }: MethodRowProps) => {
```

Comment:
Maybe that would make it easier to read?

```
if(!methods || method?.length === 0) {return null} 
```
Then we don't have to spread the checks across JSX